### PR TITLE
added Umbraco to Swags

### PR DESCRIPTION
**Pull request by duydosomething**
- [ ] I've checked that this isn't a new swag opportunity proposal.
- [ ] I've checked that this isn't a duplicate pull request.

Added Umbraco to Swags. I saw that Umbraco was added the previous year as well. Should we leave that one or have this new one replace the 2018 entry?

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "name": "CircleCI",
     "difficulty": "medium",
     "description": "Participate in #Orbtoberfest with CircleCI and claim limited edition swag! 1 - 3 pull requests: Limited-Edition Sticker, 4+ pull requests: Limited-Edition T-shirt",
-    "reference": "https://hacktoberfest.circleci.com/#/" ,
```
Comment (aslafy-z) [inline data.json]: Theses changes are out of the scope of this PR, can you please revert them?

File: data.json [hunk 2]
```diff
@@ -111,7 +111,7 @@
     "name": "Hacktoberfest 2019",
     "difficulty": "hard",
     "description": "To qualify for the official limited edition Hacktoberfest T-shirt, you must register and then make four pull requests (PRs) between October 1-31 (in any time zone).",
-    "reference": "https://hacktoberfest.digitalocean.com/" ,
```
Comment (aslafy-z) [inline data.json]: Theses changes are out of the scope of this PR, can you please revert them?

Comment (duydosomething) [reply data.json]: Reverted the changes caused by my linter. Sorry!

File: data.json (hunk 1 above)
Comment (duydosomething) [reply data.json]: Reverted the changes caused by my linter. Sorry!

**Comment by crweiner** [comment]
Umbraco has asked not to be on these swags lists, you can read more and see a link to their tweets [here](https://github.com/crweiner/hacktoberfest-swag-list/issues/44#issuecomment-535131859). Personally, I don't get it. If you don't get it either, send them a tweet to ask why.

**Comment by duydosomething** [comment]
@crweiner Hm.. missed this thread comment about Umbraco. Yeah, I'm not too sure because if anything this project gives them more exposure. I don't want to go against their wishes, so feel free to close this since it looks like they are firm with their decision lol.

**Comment by crweiner** [comment]
> Yeah, I'm not too sure because if anything this project gives them more exposure.

Precisely my point to them. I don't quite understand how a company has a slice of their marketing budget devoted to swag but doesn't want any free marketing for their product...

Send them a tweet referencing that thread and ask why.

If they don't want to actually celebrate Hacktoberfest (and would rather play their own game), then I say let's not give them any publicity....which is exactly what they want.

However, this isn't my repo. That's up to people like @swapagarwal and @aslafy-z to decide. Just letting you know how they are treating people like us this year...

**Comment by vikaspotluri123** [comment]
As they haven't explicitly asked to be unlisted, I'm fine with leaving them up, and updating the offering for this year. If they want to be unlisted, they're free to contact us to be removed, as stated in the footer of our site.

**Comment by crweiner** [comment]
Absolutely, you do as you wish. I think their request is silly and rather unenforceable anyway. Like they'll take down every Reddit post? Blog post? It's kinda crazy. I say, if they want silence, so be it, but you can certainly do as however you see fit.

**Comment by swapagarwal** [comment]
Thanks @duydosomething! 👍

**Pull request merged**